docs/UserGuide.md

```diff
@@ -38,17 +52,19 @@ Before you start using **_TBM_**,
 
 1. Navigate to the directory that the application file was downloaded to. To start the application:
 
-    1. Double click on the `tbm.jar` file and the application will start shortly.
+    1. Double click on the `TBM.jar` file and the application will start shortly.
 
-    1. Alternatively, open any shell terminal in the current directory and run the command `java -jar tbm.jar`.
+    1. Alternatively, open any shell terminal in the current directory and run the command `java -jar TBM.jar`.
 
 Now that you have started **_TBM_**,
 
 1. Type any command in the command box and press Enter to execute it.
 
-1. (Recommended for new users) The [`help`](#Viewing-help-help) command will show a help page of some commonly used commands to fully utilise **_TBM_**.
+1. ***(Recommended for new users)*** The [`help`](#Viewing-help-help) command will show a help page of some commonly
+ used commands to fully utilise **_TBM_**.
 
-   * Type the **`help`** command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
+   * ***Either*** press the **F1** key on your keyboard, ***or*** type the **`help`** command in the command box and
+    press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
```

Comment:
Not sure if the triple * is intentional? But it seems to give the same result as **_, (italics + bold) which I think is clearer, since triple * and double * look too similar

Reply:
It was intentional and I think this is pretty nitty-gritty but I've changed it 👍